tailwind.config.js

```diff
@@ -0,0 +1,63 @@
+import tailwindConfig from '@hypothesis/frontend-shared/lib/tailwind.preset.js';
+
+export default {
+  presets: [tailwindConfig],
+  content: [
+    './h/static/scripts/**/*.{js,ts,tsx}',
+    './node_modules/@hypothesis/frontend-shared/lib/**/*.js',
+  ],
+  theme: {
+    extend: {
+      animation: {
```

Comment:
The `fade-in`, `fade-out` and `slide-in-from-right` animations are now part of our component library so we should be able to remove them here and in Via. I would remove everything related to animations from here in this PR. If we find there is anything missing in the shared library we can add them there. The animations are used by the `ToastMessages` component.

Reply:
Done: https://github.com/hypothesis/h/pull/8789/commits/013c74feab766e1857fb42ae5f9f54a6eadd8846